_targets.R

```diff
@@ -6,18 +6,76 @@ source("3_visualize/src/plot_timeseries.R")
 options(tidyverse.quiet = TRUE)
 tar_option_set(packages = c("tidyverse", "dataRetrieval")) # Loading tidyverse because we need dplyr, ggplot2, readr, stringr, and purrr
 
+## Define vars
+site_1 <- "01427207"
+site_2 <- '01432160'
+site_3 <- "01436690"
+site_4 <- '01466500'
+parameterCd <- '00010'
+startDate <- "2014-05-01"
+endDate <- "2015-05-01"
+fetch_out_folder <- "1_fetch/out"
```

Comment:
Added the variables here. Curious, when is it better to put these as targets vs. just global variables? My sense is that it is when I want the definition of a given variable to be set as a specific target and, is therefore an important component of the workflow. Interested in reviewer's take on this, and a real example

Reply:
Good question. I don't think that there's any hard and fast rule about this. It certainly wouldn't be wrong to use targets instead of variables here - it would work just fine. But generally I would favor targets for things which are doing some work, rather than configuration variables. It's just nice and readable to have all of the configuration variables for a pipeline together in one place, and doing it with variables is more concise and easier to scan than doing it with targets.